pkg/storage/cache_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package storage_test
+
+import (
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/storage/leveldbstore"
+	"github.com/ethersphere/bee/pkg/storage/storagetest"
+)
+
+func TestCache(t *testing.T) {
+	t.Parallel()
+
+	store, err := leveldbstore.New(t.TempDir(), nil)
+	if err != nil {
+		t.Fatalf("create store failed: %v", err)
+	}
+
+	s := storage.NewCacheLayer(store, 100_000)
+	t.Cleanup(func() { _ = s.Close() })
```

Comment:
This is not necessary since the `storagetest.TestStore` also tests the `Close` method call.